msteams-platform/TOC.yml

```diff
@@ -775,6 +775,14 @@
       - name: Teams message extension vs. Outlook add-in
         href: m365-apps/me-or-outlook-add-in.md
         displayName: mail, Microsoft 365, Office 365, Outlook add-in, message extension
+  - name: Adaptive Card-based Loop components
+    items:
+      - name: Loop component design guidelines
+        href: m365-apps/design-loop-components.md
+        displayName: loop component, weburl, metadata
+      - name: Build Adaptive Card-based Loop component
+        href: m365-apps/cards-loop-component.md
+        displayName: loop component, weburl, metadata
```

Comment:
Let's move this before "Loop component design guidelines"

Reply:
Hi @shuchauh,

As per our doc guidelines, we must have the design guidelines first and then have the feature related info.

Thanks,
Yash

Reply:
Updated the title to "Design your Loop component".